### PR TITLE
Updating pcs_status class to show bad nodes, aka nodes that

### PR DESCRIPTION
**Pull request by wduffeeb**
Updating pcs_status class to show bad nodes, aka nodes that show as online but are not in a clean status when the pcs_status output is dug into.

### CODE REVIEW

File: insights/parsers/pcs_status.py [hunk 1]
```diff
@@ -128,10 +129,15 @@ def parse_content(self, content):
                 else:
                     self.data["WARNING"] = [line.strip()]
                 continue
+            if (line.startswith("RemoteNode") or line.startswith("GuestNode") or line.startswith("Node")):
+                linesplit = line.split()
+                bad_node = {}
+                bad_node['name'] = linesplit[1][0:-1]
+                bad_node['status'] = line.split(':')[1][1:]
+                self.bad_nodes.append(bad_node)
```
Comment (bfahr) [inline insights/parsers/pcs_status.py]: @wduffeeb do you need to add some tests for this?

Comment (wduffeeb) [reply insights/parsers/pcs_status.py]: @bfahr test update committed

**Comment by wduffeeb** [comment]
Test submitted. Flake8 checks passed.

**Comment by xiangce** [comment]
@wduffeeb - Would you please to submit the corresponding PR to the 1.x branch? Then I will merge them together.

**Comment by wduffeeb** [comment]
Hi @xiangce @bfahr - how do I submit a PR to a specific branch?

**Comment by kylape** [comment]
@wduffeeb I suggest you take a look at the pull request documentation from github regarding [changing the base branch](https://help.github.com/articles/creating-a-pull-request/#changing-the-branch-range-and-destination-repository).  Basically, when you're opening a pull request, there's a drop down where you can choose the base repository and branch.  You simply choose 1.x from that drop down.

**Comment by xiangce** [comment]
@wduffeeb - I created it (#394) by using the following command:

~~~
git checkout master && git pull --rebase <---- To confirm local master it the latest
git checkout pcs_update_badnodes
git rebase master
git format-patch master                          <--- generate patches
git checkout -b wduffeeb-pcs_update_badnodes_1.x
git am *.patch                                         <--- apply above patches to 1.x
git push origin wduffeeb-pcs_update_badnodes_1.x 
~~~

**Comment by wduffeeb** [comment]
Thanks @xiangce - I appreciate you taking this. I don't know that I would have ever figured that out.

**Pull request merged**